cmake/modules/FindLibBot.cmake

```diff
@@ -7,14 +7,21 @@
 # LIBBOT_INCLUDE_DIRS
 # LIBBOT_LIBRARIES
 
+find_package(GTK2 REQUIRED gtk)
+
+find_package(libbot QUIET)
```

Comment:
i think that this new code probably isn't actually being tested on travis-ci?  We do have a libbot build on travis-ci, but it's not compiled by drake.